app.js

```diff
@@ -71,7 +71,8 @@ app.events = new EventEmitter()
 // view engine setup
 app.set('views', fsPath.join(__dirname, 'views'));
 
-var templateCache = new TemplateCache()
+// global as graphController may want to recompile
```

Comment:
you could export a singleton with `module.exports` from templateCache.js instead of having a global